package.json

```diff
@@ -40,17 +40,17 @@
   },
   "devDependencies": {
     "@backstage/cli": "0.25.2",
-    "@smithy/util-utf8": "2.1.1",
+    "@smithy/util-utf8": "2.3.0",
```

Comment:
in 1.1.x we had to specifically pin 2.11 https://github.com/janus-idp/backstage-showcase/blob/1.1.x/package.json#L43-L45 to make Cachito happy. 

This might break downstream builds, if we don't also add the other deps explicitly

```
    "@smithy/is-array-buffer": "2.2.0",
    "@smithy/util-buffer-from": "2.2.0",
```

Reply:
https://gitlab.cee.redhat.com/rhidp/rhdh/-/pipelines/7867868 running

Reply:
looks OK in registry-proxy.engineering.redhat.com/rh-osbs/rhdh-rhdh-hub-rhel9:1.2-40